src/test/java/org/broadinstitute/consent/http/service/DataAccessReportsParserTest.java

```diff
@@ -18,27 +13,28 @@
 import java.util.Iterator;
 import java.util.List;
 import java.util.stream.Stream;
-
-import static org.junit.Assert.assertTrue;
+import org.broadinstitute.consent.http.enumeration.HeaderDAR;
+import org.broadinstitute.consent.http.models.DataAccessRequest;
+import org.broadinstitute.consent.http.models.DataAccessRequestData;
+import org.broadinstitute.consent.http.models.DatasetDetailEntry;
+import org.broadinstitute.consent.http.models.Election;
+import org.junit.Test;
 
 public class DataAccessReportsParserTest {
 
     DataAccessReportsParser parser;
-    private final String SC_ID = "SC-01253";
     private final String CONSENT_NAME = "ORSP-1903";
     private final String NAME = "Test";
-    private final String REQUESTER = "Wesley";
-    private final String ORGANIZATION = "Broad";
     private final String RUS_SUMMARY = "Purpose";
-    private final String sDUL = "Samples Restricted for use with \"cancer\" [DOID_162(CC)]\n" +
-            "Future use by for-profit entities is prohibited [NPU]\n" +
-            "Future use of aggregate-level data for general research purposes is prohibited [NPNV]\n" +
-            "Notes:\n" +
-            "Future use for methods research (analytic/software/technology development) is not prohibited\n" +
-            "Future use as a control set for diseases other than those specified is not prohibited";
+    private final String sDUL = """
+        Samples Restricted for use with "cancer" [DOID_162(CC)]
+        Future use by for-profit entities is prohibited [NPU]
+        Future use of aggregate-level data for general research purposes is prohibited [NPNV]
+        Notes:
+        Future use for methods research (analytic/software/technology development) is not prohibited
+        Future use as a control set for diseases other than those specified is not prohibited""";
```

Comment:
nit: """ should be on newline

Reply:
also: unrelated, but should `not prohibited` be `allowed`? confusing verbage

Reply:
> also: unrelated, but should `not prohibited` be `allowed`? confusing verbage

Language choices here are unrelated to the PR. However, these are also intentional. I believe I have made the same arguments before. [See this search for examples](https://github.com/DataBiosphere/consent-ontology/search?q=%22Future+use+as+a+control+set%22) of why this test does what it does.